### PR TITLE
fabric: add controller mode label

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: cluster/manifests/fabric-gateway/deployment.yaml [hunk 1]
```diff
@@ -26,7 +27,8 @@ spec:
         deployment: fabric-gateway-controller
         application: gateway-operator
         component: fabric-gateway-controller
-        version: {{ $version }}
+        version: "{{ $version }}"
+        mode: "{{ .ConfigItems.fabric_gateway_controller_mode }}"
```
Comment (szuecs) [inline cluster/manifests/fabric-gateway/deployment.yaml:31]: does empty string work?

Comment (szuecs) [reply cluster/manifests/fabric-gateway/deployment.yaml:31]: if not maybe prefix with single character or so like
```
mode: "_{{ .ConfigItems.fabric_gateway_controller_mode }}"
```

Comment (AlexanderYastrebov) [reply cluster/manifests/fabric-gateway/deployment.yaml:31]: `fabric_gateway_controller_mode` is `disabled` by default

Comment (AlexanderYastrebov) [reply cluster/manifests/fabric-gateway/deployment.yaml:31]: https://github.com/zalando-incubator/kubernetes-on-aws/blob/061825f64cce867a0b414d5e4d07e449ff781ab8/cluster/config-defaults.yaml#L265-L273

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**